Xwt/Xwt.Drawing/Image.cs

```diff
@@ -1183,16 +1184,22 @@ public override object LoadImage (string fileName)
 
 		public override IEnumerable<string> GetAlternativeFiles (string fileName, string baseName, string ext)
 		{
-			if (!Context.RegisteredStyles.Any ()) {
```

Comment:
This check breaks my brain. I don't really understand why it is `!Context.RegisteredStyles.Any()`